sktime/forecasting/chronos.py

```diff
@@ -0,0 +1,161 @@
+    """
+
+    # tag values are "safe defaults" which can usually be left as-is
+    _tags = {
+        "python_dependencies": ["torch"],
+        "y_inner_mtype": "pd.Series",
+        "X_inner_mtype": "pd.DataFrame",
+        "scitype:y": "univariate",
+        "ignores-exogeneous-X": True,
+        "requires-fh-in-fit": False,
+        "authors": ["RigvedManoj"],
+        "maintainers": ["RigvedManoj"],
+    }
+
+    def __init__(self, modelName, num_samples=20, temperature=1.0, top_k=50, top_p=1.0, args_list=None, kwargs_dict=None):
+        self.modelName = modelName
+        self.num_samples = num_samples
+        self.temperature = temperature
+        self.top_k = top_k
+        self.top_p = top_p
+        self.args_list = args_list
+        self.kwargs_dict = kwargs_dict
+
+        super().__init__()
+
+    def _fit(self, y, X=None, fh=None):
+        """Fit forecaster to training data.
+
+        private _fit containing the core logic, called from fit
+
+        Writes to self:
+            Sets fitted model attributes ending in "_".
+
+        Parameters
+        ----------
+        y : pd.Series
+            Target time series to which to fit the forecaster.
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored.
+
+        Returns
+        -------
+        self : reference to self
+        """
+        import torch
+        from chronos import ChronosPipeline
+        if self.args_list is not None:
+            args_list = [self.modelName] + self.args_list
+        else:
+            args_list = [self.modelName]
+        if self.kwargs_dict is None:
+            self._model = ChronosPipeline.from_pretrained(*args_list)
+        else:
+            self._model = ChronosPipeline.from_pretrained(*args_list, **self.kwargs_dict)
+        self._context = torch.tensor(y.values)
+        return self
+
+    def _predict(self, fh=None, X=None):
+        """Forecast time series at future horizon.
+
+        private _predict containing the core logic, called from predict
+
+        Parameters
+        ----------
+        fh : guaranteed to be ForecastingHorizon or None, optional (default=None)
+            The forecasting horizon with the steps ahead to predict.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored.
+
+        Returns
+        -------
+        y_pred : pd.DataFrame
+            Predicted forecasts.
+        """
+        prediction_length = len(fh)
+        forecast = self._model.predict(
+            self._context,
+            prediction_length,
+            num_samples=self.num_samples,
+            temperature=self.temperature,
+            top_k=self.top_k,
+            top_p=self.top_p,
+        )
+        values = np.median(forecast[0].numpy(), axis=0)
```

Comment:
This is mainly a compatibility question to @fkiraly : do we intend to use predict of all forecasters as median or mean? Depending on that, this line may needs to be changed.